README.md

```diff
@@ -0,0 +1,230 @@
+# Unikraft ELF Loader
+
+## Building the ELF Loader unikernel
+
+`elfloader` can be built using the provided `Makefile`.
+
+Make sure the source code for [Unikraft](https://github.com/unikraft/unikraft), [libelf](https://github.com/unikraft/lib-libelf) and [lwip](https://github.com/unikraft/lib-lwip) is available at the paths referenced by the `UK_ROOT` and `UK_LIBS` variables respectively.
+Execute the `make menuconfig` command to enter the configuration dialog, select a target platform (e.g. "KVM"), and exit to generate the unikernel configuration.
+
+Executing `make` a second time triggers a build of the unikernel for the selected platform(s).
```

Comment:
This is not enough. There are several options that must be enabled for a proper build, that includes networking, virtual memory and others:
- The `vfscore` configuration with `fs0` is mentioned below.
- `ukmmap` must be disabled, and then `ukvmem` and `posix-mmap` must be enabled.
- Paging must be enabled in `Platform Configuration` -> `Platform Interface Options` -> `Virtual memory API`.
- `Virtio PCI device support` must be enabled in `Platform Configuration` -> `KVM Guest` -> `Virtio`
- Useful internal libraries such as `posix-event`, `posix-futex` must be enabled.

Reply:
Good point to mention it here. The PR is already changing a lot of the config defaults with `imply` options, but you are right 9pfs is not covered if "Virtio-PCI" is not enabled and paging must be enabled by hand - although, this could be done with an imply, as well. I'll mention these settings :+1: